openmc/filter.py

```diff
@@ -1324,13 +1323,11 @@ def get_pandas_dataframe(self, data_size, **kwargs):
         # Extract the lower and upper angle bounds, then repeat and tile
         # them as necessary to account for other filters.
         lo_bins = np.repeat(self.bins[:-1], self.stride)
-        hi_bins = np.repeat(self.bins[1:], self.stride)
         tile_factor = data_size / len(lo_bins)
         lo_bins = np.tile(lo_bins, tile_factor)
-        hi_bins = np.tile(hi_bins, tile_factor)
 
         # Add the new angle columns to the DataFrame.
-        df.loc[:, self.type + ' low'] = lo_bins
+        df.loc[:, 'polar low'] = lo_bins
```

Comment:
It looks like an oversight that there are no `polar high` and `azimuthal high` columns.  Please add those the the DataFrame